src/mixins/common-mixin.ts

```diff
@@ -51,4 +52,24 @@ export default class CommonMixin extends Vue {
     }
     return true
   }
+
+  /**
+   * Extracts validation flags from validation item detail object
+   * @param validation items details object
+   * @return array of validation flags for all validation items
+   */
+  buildValidFlags (validationItems: ValidationItemDetailIF[]): object {
+    let result = {}
+    for (const vi of validationItems) { result[vi.name] = vi.valid }
+    return result
```

Comment:
This function looks fine except for the return type: it's not a complete object since (I think) it doesn't contain certain object methods. I think it would be better to return type `any`. Thoughts?

Reply:
(Line 62 declares an empty object so I might be wrong here.)

Reply:
yes, `any` makes sense.  will get that updated.

Reply:
taking another look based on your last comment...

Reply:
so it should be a complete object as you its initialized using `let result = {}` as you had noted.